contracts/src/v0.8/automation/AutomationBase.sol

```diff
@@ -8,8 +8,8 @@ contract AutomationBase {
    * @notice method that allows it to be simulated via eth_call by checking that
    * the sender is the zero address.
    */
-  function preventExecution() internal view {
-    if (tx.origin != address(0)) {
```

Comment:
This changes functionality, be very very sure this is a change you want to make. You can always ignore solhint rules with `// solhint-disable <rule_name>`

Reply:
You are right, best to just ignore to be safe.  Tagging @RyanRHall to confirm. I will update later.